Description: Use dpkg-buildflags.

* Use the flags from CFLAGS, CPPFLAGS and LDFLAGS. They are set in the rules file.

Origin: http://anonscm.debian.org/gitweb/?p=pkg-games/bsnes.git;a=tree;f=debian/patches

--- a/Makefile
+++ b/Makefile
@@ -13,8 +13,8 @@
 # console := true
 
 # compiler
-flags   += -I. -O3 -fomit-frame-pointer
-link    +=
+flags   += -I. -O3 -fomit-frame-pointer $(CFLAGS) $(CPPFLAGS)
+link    += $(LDFLAGS)
 objects := libco
 
 # profile-guided optimization mode
--- a/ananke/Makefile
+++ b/ananke/Makefile
@@ -2,7 +2,7 @@
 include ../phoenix/Makefile
 
 path := /usr/local/lib
-flags := $(flags) -O3 -fomit-frame-pointer -I..
+flags := $(flags) -O3 -fomit-frame-pointer $(CFLAGS) $(CPPFLAGS) -I..
 
 all:
 	$(compiler) $(cppflags) $(flags) -fPIC -o obj/ananke.o -c ananke.cpp
